doc/source/fake-autoscaler.rst

```diff
@@ -194,11 +194,17 @@ For the Ray OSS Buildkite environment, we thus set some environment variables:
   variable.
 
 Lastly, docker-compose obviously requires a docker image. The default docker image is ``rayproject/ray:nightly``.
-However, on our Ray OSS Buildkite environment, we want to test changes from the respective PR or Branch. Thus, we set
+The docker image requires ``openssh-server`` to be installed and enabled. In Buildkite we build a new image from
+``rayproject/ray:nightly-py37-cpu`` to avoid installing this on the fly for every node (which is the default way).
```

Comment:
for all supported python version?

Reply:
Multinode tests on buildkite only run on Python 3.7 (as do most Tune tests). The build script supports all base images though.